src/uwtools/config.py

```diff
@@ -51,115 +54,84 @@
 
 class Config(ABC, UserDict):
     """
-    This base class provides the interface to methods used to read in several configuration file
-    types and manipulate them as such.
-
-    Attributes
-    ----------
-
-    config_path
-        The file path to the configuration file to be parsed.
-
-    Methods
-    -------
-
-    _load()
-        Abstract method used as an interface to load a config file
-
-    dump_file(output_path)
-        Abstract method used as an interface to write a file to disk
-
-    from_ordereddict(in_dict)
-        Given a dictionary, replaces instances of OrderedDict with a regular dictionary
-
-    parse_include()
-        Traverses the dictionary treating the !INCLUDE path the same as
-        is done by pyyaml.
-
-    replace_templates()
-        Traverses the dictionary and renders any Jinja2 templated
-        fields.
-
-    update_values()
-        Update the values in an existing dictionary with values providedv by a second dictionary.
-        Keep any of the values that are not present in the second dictionary.
+    A base class specifying (and partially implementing) methods to read, manipulate, and write
+    several configuration-file formats.
```

Comment:
If we should stick with the NumPy style, we'll need to remove `docformatter` from the tooling, as it isn't compatible with that style, and continue to manually format our docstrings. Please let me know and I'll update appropriately -- easily done. (I'm not sure if a potential response from Gillian in our Slack conversation about documentation would make affect your decision, but FYI that's still out there.)

Reply:
I think I'm likely fine with moving away from the NumPy style, we just need to remove that from the docs. Especially if we can't control it with a tool. That's a much more important deciding factor so that we don't end up with our docs being all over the place.

I just took a look at what happens when we call `help` on this before and after this change. It's still quite readable and fills in all the info about all the methods and attributes we have documented. I'm totally cool with that, as it was essentially what we wanted to capture with the NumPy style choice, anyway. 

The ultimate goal is to give users a professional looking, helpful amount of information, and this does that!

Reply:
In that case, we could go with what's here, see if Gillian offers useful feedback on what I showed her that might make us reconsider and, if not, adopt this simple PEP 257 + Sphinx format, document that decision, and then chip away over time at bringing the larger codebase into line. And I can remove the reference to the NumPy style for now. Does that sound good?